pkg/kontainer-engine/drivers/aks/aks_driver.go

```diff
@@ -775,6 +773,7 @@ func (d *Driver) createOrUpdate(ctx context.Context, options *types.DriverOption
 		}
 	}
 
+	networkProfile.LoadBalancerSku = containerservice.LoadBalancerSku(driverState.LoadBalancerSku)
```

Comment:
How does this react with older clusters that didn't have one set ? Does it just stay blank and nothing happens.

Reply:
I have not tested this on older clusters, but this functionality should only be exposed during cluster creation. AKS only allows the standard LB to be configured at creation time.

Reply:
I think we could add a check to ensure that the LoadBalancerSku is only being configured during cluster creation.
```go
// Only configure the LoadBalancerSku during cluster creation.
if sendRBAC {
	networkProfile.LoadBalancerSku = containerservice.LoadBalancerSku(driverState.LoadBalancerSku)
}
```

Reply:
New idea: we always set the Value to "standard" currently. I can make it so that the default is `Value: ""` , and then we can have logic to set it like so...
```
if driverState.LoadBalancerSku != "" {
    	networkProfile.LoadBalancerSku = containerservice.LoadBalancerSku(driverState.LoadBalancerSku)
}
```

Reply:
Going with this change for the time being since it strikes a balance between being backwards-compatible and non-intrusive.

Reply:
Undid this as it was unnecessary. The pre-existing clusters were unaffected by the new change.